src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -1121,6 +1121,21 @@ describe('Step Details', () => {
     await user.type(nameInput, 'valid-name');
     expect(getNextButton()).not.toHaveClass('pf-m-disabled');
     expect(getNextButton()).toBeEnabled();
+
+    // Enter description image
+    const descriptionInput = screen.getByRole('textbox', {
+      name: /description 0\/250/i,
+    });
+    // 101 character name
```

Comment:
I think this comment is from the image name, not description... Maybe just delete it, it is obvious what is happening in the line below...

Reply:
removed. thank you